lib/features/tracker/ui/containers/workout_summary_header.dart

```diff
@@ -13,9 +14,10 @@ class WorkoutSummaryHeader extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final workout = ref.watch(liveWorkoutControllerProvider);
+    final weightUnit = ref.watch(weightUnitControllerProvider);
     final totalVolume = ref
         .watch(liveWorkoutControllerProvider.notifier)
-        .getTotalVolume(Units.lbs);
+        .getTotalVolume(Units.values[weightUnit.index]);
```

Comment:
We can just pass in the Units enum here instead of using the index.

Also why did you create a secondary WeightUnits enum instead of using the existing Units enum?

Reply:
Changed to strickly use Units enum